src/main/resources/view/SummaryListPanel.fxml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.layout.VBox?>
+
+<VBox xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Not sure if you missed this line but xmlns="http://javafx.com/javafx8" is xmlns="http://javafx.com/javafx11" in the other .fxml files